app/containers/App/Application.js

```diff
@@ -368,3 +423,25 @@ export default connect(
   mapStateToProps,
   mapDispatchToProps
 )(Application);
+
+const SessionExpiredNotification = () => {
+  const [time, setTime] = useState(9);
```

Comment:
nit:
don't use constants in code like this.  Define it as a constant at the top of the file or in a constants.js file and reference it here.  

why?
Doing something like
SESSION_EXPIRATION_TIMEOUT =  9
will be really helpful for anyone reading your code.